client/src/utils/distance.js

```diff
@@ -22,16 +22,16 @@ function getCurrentLocation(options) {
     );
   });
 }
+
 export async function inOutLocation() {
-  try {
-    return await getCurrentLocation({
-      enableHighAccuracy: true,
-      timeout: 5000,
-      maximumAge: 0
-    });
-  } catch (e) {
-    if (e.name === 'PositionError') {
-      console.log(e.message + '. code = ' + e.code);
-    }
-  }
-}
+  return await getCurrentLocation({
+    enableHighAccuracy: true,
+    timeout: 5000,
+    maximumAge: 0
+  })
+    .catch(e => {
+      if (e.name === 'PositionError') {
+        console.log(e.message + '. code = ' + e.code);
```

Comment:
I probably wrote this one in the past, but would be good to update to a `console.error()` and return a value so we don't break the ui in a position error.

Reply:
Done.